src/components/Wrapper/index.js

```diff
@@ -1,12 +1,15 @@
 import classNames from 'classnames';
+import routeMap from '../../utils/routeMap';
+
 
 import wrapperStyle from './style.scss';
 
 const Wrapper = ({
-  className, children, id, isHome,
+  className, children, id, isHome, location,
 }) => {
   const wrapperClasses = classNames(wrapperStyle.wrapper, className, {
     [wrapperStyle.isHome]: isHome,
+    [wrapperStyle.isWhite]: location === routeMap.no_place_left,
```

Comment:
Try not to use snake case keys. Try to change the naming `no_place_left`

Reply:
👍